web/scm/src/styles.scss

```diff
@@ -1 +1,3 @@
-/* You can add global styles to this file, and also import other style files */
+body {
+    margin: 0;
+}
```

Comment:
we don't have this check here, but it's widely used across google -> Having blank new line at the end of the all text files

Reply:
Also PTAL here https://github.com/google/gts

Reply:
Got it. Will follow the style guide